ui/tests/login-first-time.spec.ts

```diff
@@ -0,0 +1,17 @@
+import { test, expect } from "@playwright/test";
+import { setupTotp } from "./helpers/totp";
+import { startGrafanaNewUserFlow } from "./helpers/grafana";
+import { resetIdentities } from "./helpers/kratosIdentities";
+import { userPassLogin } from "./helpers/login";
+
+test("first time login to grafana", async ({ context, page }) => {
+  resetIdentities();
+  await startGrafanaNewUserFlow(page);
+  await userPassLogin(page);
+  await setupTotp(context, page);
+
+  await expect(page.getByText("Account setup complete")).toBeVisible();
+  await expect(page).toHaveScreenshot({ fullPage: true });
```

Comment:
opinion: this is interesting `toHaveScreenshot` seems like it could be "brittle", I saw we can update screenshots automatically with Playwright. I'd say let's keep this in, but let's be mindful of this function for future tests

Reply:
I agree. I was hesitant to include it or not. After considering the option carefully, I think the benefits outweigh not having it:
 - have clear documentation of how the screens look like in the repo.
 - text assertions with `expect(...)` only cover partially the functionality available in a given screen. We had bugs in the past, that were due to exposure of additional functionality in screens, where it should be hidden. Having screenshots seems like the right approach against this class of bugs.
 - easily update the screens when updating 3rd party like vanilla changes layout just so slightly. As you mentioned already.
 - see in a PR when screens change, as that PR should contain the old and the new screenshot

Reply:
It makes perfect sense, thank you for explaining this!